scripts/bootstrap.ps1

```diff
@@ -0,0 +1,20 @@
+param(
+    [switch]$build = $false
+)
+
+# Switch to TigerBeetle directory.
+cd "$PSScriptRoot\.."
+
+git fetch --tags --force --quiet
+
+$version = $( git tag --sort=committerdate | Select-Object -Last 1 )
+
+if ($build) {
+    git checkout $version
```

Comment:
I wonder if this would break if `git checkout` overwrites bootstrap.ps1. Windows doesn't like replacing file which is being executed (but maybe this is relaxed for scripts which are not executed themselves?)

Reply:
Yes it would probably crash in that scenario. But our bootstrap script wouldn't work already if you copied and pasted it since the directory would also already exist. So the one-liner is really only meant as a one-time thing. Right?

Reply:
>So the one-liner is really only meant as a one-time thing. Right?

The one-liner itself does checkout. So what I am thinking here is the case where we update script on master, relative to the tagged version. So the one-liner will start executing master version, but then overwrite it with the tagged version.

Reply:
While I've seen something along the lines of what you are talking about... I can't remember the rules. Can't delete while open for sure. But it seems like this works? Or what am I forgetting?

![image](https://github.com/tigerbeetledb/tigerbeetle/assets/3925912/04777251-7920-4355-92fc-ae7f80b140df)

Reply:
It seems like on Windows it might be completely fine, but a problem on Linux... https://stackoverflow.com/questions/60692684/edit-powershell-script-while-its-running

(I've seen super weird behaviour first hand, editing shell scripts while they're running)